### DISCUSSION TITLE
Getting Started with Seeed Studio XIAO RP2350 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1599] [Q&A]
# Getting Started with Seeed Studio XIAO RP2350 | Seeed Studio Wiki

XIAO RP2350, a cutting-edge microcontroller from Seeed Studio. It features a dual-core processor, increased SRAM and flash memory, and enhanced connectivity.

https://wiki.seeedstudio.com/getting-started-xiao-rp2350/

**Comment by leonheller** [comment]
Omitted to mention that the RP2350 also has two RISC-V cores. Only two cores can be used at once.
